internal/config/config.go

```diff
@@ -229,26 +224,23 @@ func (o *Override) Parse() error {
 		if lastSlash == -1 {
 			return fmt.Errorf("Package override `go_type` specifier %q is not the proper format, expected 'package.type', e.g. 'github.com/segmentio/ksuid.KSUID'", o.GoType)
 		}
-		typename = o.GoType[lastSlash+1:]
-		if strings.HasPrefix(typename, "go-") {
-			// a package name beginning with "go-" will give syntax errors in
-			// generated code. We should do the right thing and get the actual
-			// import name, but in lieu of that, stripping the leading "go-" may get
-			// us what we want.
-			typename = typename[len("go-"):]
-		}
-		if strings.HasSuffix(typename, "-go") {
-			typename = typename[:len(typename)-len("-go")]
+
+		importpath := o.GoType[:lastDot]
+		typename := o.GoType[lastDot+1:]
+
+		// Load given import path to ensure validity
+		p, err := packages.Load(&packages.Config{Mode: packages.NeedName}, importpath)
```

Comment:
I believe `packages.Load` required the Go tooling to be installed on the host, a dependency I'm not willing to take on right now. sqlc should work without any additional software. I'm investigating if there's parts of the code base that we can use to parse the import path without having to verify that it's valid.